templates/status_email@.service.erb

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Status email for %i to <%= @user %>
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/systemd-email <%= @user =~ /@/ ? @user : "#{@user}@#{@facts['networking']['fqdn']}" %> %i
```

Comment:
According to `man 5 systemd.unit` you can use `%H` for the long hostname which I think should be preferred over facts.

Reply:
good point